docs/source/Recipes/Appetizers/Dips_Hummus.rst

```diff
@@ -1,5 +1,64 @@
 .. index::
    single: dips; hummus
 
-Hummus
+To make hummus
 =====================
+
+Hummus is a glorious dip, which is well suited to dehydration and backpacking before the addition of oil. It will save any lunch.
+
+.. makes:: 2 portions
+
+.. ingredients::
+
+   - 250g cooked chickpeas
+   - 1tsp tahini
+   - 1/2 lemon for its juice
+   - fresh garlic
+   - salt
+   - pepper
+   - sumac
+   - olive oil
+
+.. procedure::
+
+   Chickpeas
+   -----------------
```

Comment:
I don't think sections in the procedure are appropriate. I'd rather have a single block with how to cook chickpeas and how to make hummus, and add the tahini as a separate recipe (and rest file) and link it from within the procedure.

Reply:
(See https://github.com/carlogico/Recipes/pull/15/checks#step:4:110)